### ISSUE TITLE
assistsnce

### ISSUE THREAD

**Issue by Crypto-bitcoin** [w3c/wcag3#118]
i am not able to use my own funds what is the point of crypto if i cannot use it

**Comment by Crypto-bitcoin** [comment]
why was this closed

**Comment by dsommers** [comment]
@Crypto-bitcoin A very strong hint here:

> W3C Accessibility Guidelines (WCAG) 3.0 will provide a wide range of recommendations for making web content more accessible to users with disabilities.

Source: https://github.com/w3c/wcag3/blob/main/guidelines/index.html
